Monika After Story/game/script-affection.rpy

```diff
@@ -300,6 +300,7 @@ init 15 python in mas_affection:
         """
         Runs when transitioning from distressed to broken
         """
+        evhand._unlockEventLabel("monika_song_crime_and_punishment")
```

Comment:
why are you not locking the topic when going from broken to distressed?

Reply:
I felt that this could serve as a reminder of how the player treated Monika before building back up affection. 

Similar to mas_affection_apology, it sticks and still has some sort of effect if the player went through it(assuming that's actually how it works and it doesn't get forgotten), but it's not exactly something to be proud of or something you would show off, considering what you have to do to get it. 

If it's still a problem, the song could be locked when going from broken to distressed.